frontend/web/components/import-export/ImportPage.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { FC, useEffect, useState } from 'react'
+import _data from 'common/data/base/_data'
+import {
+  useCreateLaunchDarklyProjectImportMutation,
+  useGetLaunchDarklyProjectImportQuery,
+} from 'common/services/useLaunchDarklyProjectImport'
+import AppLoader from 'components/AppLoader'
+import InfoMessage from 'components/InfoMessage'
+import Tabs from 'components/base/forms/Tabs'
+import Input from 'components/base/forms/Input'
+import Utils from 'common/utils/utils'
+import Button from 'components/base/forms/Button'
+import PanelSearch from 'components/PanelSearch'
+import TabItem from 'components/base/forms/TabItem'
+import FeatureExport from './FeatureExport'
+import { createFeatureExport } from 'common/services/useFeatureExport'
+import { getStore } from 'common/store'
+import FeatureImport from './FeatureImport'
+import AccountStore from 'common/stores/account-store'
+import Constants from 'common/constants'
+
+type ImportPageType = {
+  projectId: string
+  projectName: string
+}
+
+const ImportPage: FC<ImportPageType> = ({ projectId, projectName }) => {
+  const [tab, setTab] = useState(0)
+  const [LDKey, setLDKey] = useState<string>('')
+  const [importId, setImportId] = useState<string>('')
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+  const [isAppLoading, setAppIsLoading] = useState<boolean>(false)
+  const [projects, setProjects] = useState<string>([])
+  const [createLaunchDarklyProjectImport, { data, isSuccess }] =
+    useCreateLaunchDarklyProjectImportMutation()
+
+  const hasFlagsmithImport = Utils.getFlagsmithHasFeature(
+    'flagsmith_import_export',
+  )
+
+  const {
+    data: status,
+    isSuccess: statusLoaded,
+    refetch,
+  } = useGetLaunchDarklyProjectImportQuery(
+    {
+      import_id: importId,
+      project_id: projectId,
+    },
+    { skip: !importId },
+  )
+
+  useEffect(() => {
+    const checkImportStatus = async () => {
+      setAppIsLoading(true)
+      const intervalId = setInterval(async () => {
+        await refetch()
+
+        if (statusLoaded && status && status.status.result === 'success') {
+          clearInterval(intervalId)
+          setAppIsLoading(false)
+          window.location.reload()
+        }
+      }, 1000)
+    }
+
+    if (statusLoaded) {
+      checkImportStatus()
+    }
+  }, [statusLoaded, status, refetch])
+
+  useEffect(() => {
+    if (isSuccess) {
+      setImportId(data.id)
+      refetch()
+    }
+  }, [isSuccess, data, refetch])
+
+  const getProjectList = (LDKey: string) => {
+    setIsLoading(true)
+    _data
+      .get(`https://app.launchdarkly.com/api/v2/projects`, '', {
+        'Authorization': LDKey,
+      })
```

Comment:
Here I'm a little confused why LD came into this feature, but it's on the same import page so maybe that is relevant?

Reply:
It hasn't the file was just moved and the git diff here things it's all new code.